src/adapter/src/active_compute_sink.rs

```diff
@@ -94,6 +95,8 @@ pub enum ActiveComputeSinkRetireReason {
 pub struct ActiveSubscribe {
     /// The ID of the connection which created the subscribe.
     pub conn_id: ConnectionId,
+    /// The UUID of the connection which created the subscribe.
+    pub conn_uuid: Uuid,
```

Comment:
Can we call this `session_uuid`? Internally I think it would be good to try to be as clear as possible about the difference between a "connection ID" and a "session (UU)ID".